client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -67,18 +67,13 @@ const RobotOverview = () => {
    */
   const createRobotBoxes = (searchValue2) => {
     const filteredBotList = Object.values(robotList).filter((val) =>
-      val.robotName
-        .toUpperCase()
-        .includes(searchValue2.toUpperCase())
+      val.robotName.toUpperCase().includes(searchValue2.toUpperCase())
     );
 
     return (
       <>
         {filteredBotList.map((val) => (
-          <RobotContainer 
-            robotId={val.robotId} 
-            robotName={val.robotName} 
-          />
+          <RobotContainer robotId={val['_id']} robotName={val.robotName} />
```

Comment:
ESLINT says, that you should use the dot notation here `robotId={val.id}`

Reply:
@WolfgangDaniel the field is called _id in MongoDB but it is possible to access that field through calling it by just id. I learned this just recently, but that being said, the linter is probably right :)

Reply:
✅